examples/mutators/apply-setters/simple/README.md

```diff
@@ -55,17 +55,17 @@ kind: MyKind
 metadata:
   name: ns
 environments: # kpt-set: ${env}
-- prod
-- stage
+  - prod
+  - stage
 ```
 
 ## Function invocation
 
 Get the config example and try it out by running the following commands:
 
 ```sh
-kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/mutators/apply-setters/simple .
-kpt fn run simple
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/mutators/apply-setters/simple
```

Comment:
This path needs to be updated. `mutators` and `validators` directories are gone.

Reply:
Done.